Bottom: 4ee7776c8618603fae7b645da48ef4363e5fc12f
Top:    d6abc36565981fa2846a35ed6055d9133f877bac
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-26 14:52:05 +0000

Refresh of remove-spaces-from-step-output

---

diff --git a/src/hid/step/step_writer.c b/src/hid/step/step_writer.c
index 3e0fadc..178d8f3 100644
--- a/src/hid/step/step_writer.c
+++ b/src/hid/step/step_writer.c
@@ -85,7 +85,7 @@ make_3d_metric_step_geometric_representation_context (step_file *file)
   fprintf (file->f, "#%i=(LENGTH_UNIT()NAMED_UNIT(*)SI_UNIT(.MILLI.,.METRE.));\n", file->next_id);
   fprintf (file->f, "#%i=(NAMED_UNIT(*)PLANE_ANGLE_UNIT()SI_UNIT($,.RADIAN.));\n", file->next_id + 1);
   fprintf (file->f, "#%i=(NAMED_UNIT(*)SI_UNIT($,.STERADIAN.)SOLID_ANGLE_UNIT());\n", file->next_id + 2);
-  fprintf (file->f, "#%i=UNCERTAINTY_MEASURE_WITH_UNIT(LENGTH_MEASURE(1.0E-005),#%i, 'distance_accuracy_value', 'NONE');\n", file->next_id + 3, file->next_id);
+  fprintf (file->f, "#%i=UNCERTAINTY_MEASURE_WITH_UNIT(LENGTH_MEASURE(1.0E-005),#%i,'distance_accuracy_value', 'NONE');\n", file->next_id + 3, file->next_id);
 
   fprintf (file->f, "#%i=(GEOMETRIC_REPRESENTATION_CONTEXT(3)"
                           "GLOBAL_UNCERTAINTY_ASSIGNED_CONTEXT((#%i))"
@@ -189,7 +189,7 @@ step_product_definition_shape (step_file *file, char *name, char *description, s
 step_id
 step_cartesian_point (step_file *file, char *name, double x, double y, double z)
 {
-  fprintf (file->f, "#%i = CARTESIAN_POINT ( '%s', ( %f, %f, %f ) ) ;\n",
+  fprintf (file->f, "#%i=CARTESIAN_POINT('%s',(%f,%f,%f));\n",
                     file->next_id, name, x, y, z);
   return file->next_id++;
 }
@@ -197,7 +197,7 @@ step_cartesian_point (step_file *file, char *name, double x, double y, double z)
 step_id
 step_direction (step_file *file, char *name, double x, double y, double z)
 {
-  fprintf (file->f, "#%i = DIRECTION ( '%s', ( %f, %f, %f ) ) ;\n",
+  fprintf (file->f, "#%i=DIRECTION('%s',(%f,%f,%f));\n",
                     file->next_id, name, x, y, z);
   return file->next_id++;
 }
@@ -205,7 +205,7 @@ step_direction (step_file *file, char *name, double x, double y, double z)
 step_id
 step_axis2_placement_3d (step_file *file, char *name, step_id location, step_id axis, step_id ref_direction)
 {
-  fprintf (file->f, "#%i = AXIS2_PLACEMENT_3D ( '%s', #%i, #%i, #%i ) ;\n",
+  fprintf (file->f, "#%i=AXIS2_PLACEMENT_3D('%s',#%i,#%i,#%i);\n",
                     file->next_id, name, location, axis, ref_direction);
   return file->next_id++;
 }
@@ -237,7 +237,7 @@ step_circle (step_file *file, char *name, step_id position, double radius)
 step_id
 step_vector (step_file *file, char *name, step_id orientation, double magnitude)
 {
-  fprintf (file->f, "#%i = VECTOR ( '%s', #%i, %f ) ;\n",
+  fprintf (file->f, "#%i=VECTOR('%s',#%i,%f);\n",
                     file->next_id, name, orientation, magnitude);
   return file->next_id++;
 }
@@ -245,7 +245,7 @@ step_vector (step_file *file, char *name, step_id orientation, double magnitude)
 step_id
 step_line (step_file *file, char *name, step_id pnt, step_id dir)
 {
-  fprintf (file->f, "#%i = LINE ( '%s', #%i, #%i ) ;\n",
+  fprintf (file->f, "#%i=LINE('%s',#%i,#%i);\n",
                     file->next_id, name, pnt, dir);
   return file->next_id++;
 }
